### PR TITLE
Insert the bare minimum for the timeline, still needs re-factoring and testing

### PR DESCRIPTION
**Pull request by ThinkingTurtle**
A couple things need to change. 

First question I forget how we launch the website for testing, so I haven't checked if the code still works, could I get a refresher on how to launch the site?
 
Second question I'm not sure how templates work for the Icon, I just had code that builds the Icon like this 
buttonGCD.addEventListener('click', () => {
    const newImg = document.createElement('img');
    newImg.setAttribute('draggable', 'false');
    newImg.setAttribute('src', 'https://lds-img.finalfantasyxiv.com/d/49cbc217420d69d69bb1bfcf6fba10358dcd2521.png');
    const newItem = document.createElement('li');
    newItem.setAttribute('draggable', 'true');
    newItem.appendChild(newImg);
    sortableList.appendChild(newItem);
  }
);
Is there a way to do this with just a template or is the template more for documentation?

Things to do:
I have to make sure the timeline generates properly after I answer the above questions
and I have to move the drag drop logic from the main js file to the Icon js files. 

Html for testing if the buttons don't work:
	<li draggable="true">
         Skill
      </li>
	<li draggable="true">
        Skill
      </li>
	<li draggable="true">
        Skill
      </li>

### CODE REVIEW

File: src/modules/xiv/rotationBuilder/rotationBuilder.css [hunk 1]
```diff
@@ -7,4 +7,21 @@ main {
...
+}
+
+.sortable-list {
+	margin: 0 auto;
+}
+
+ul {
+  height: auto;
+  width: auto;
+	list-style-type: none;
```
Comment (gik0geck0) [inline src/modules/xiv/rotationBuilder/rotationBuilder.css]: nit: indentation

File: src/modules/xiv/rotationBuilder/rotationBuilder.html [hunk 2]
```diff
@@ -1,9 +1,23 @@
 <template>
     <main>
+        <script>/*
```
Comment (gik0geck0) [inline src/modules/xiv/rotationBuilder/rotationBuilder.html]: we don't need to include any script tags; it "just works"

File: src/modules/xiv/rotationBuilder/rotationBuilder.html [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
         <lightning-icon icon-name="action:approval" title="Approved"></lightning-icon>
         <lightning-card icon-name="utility:connected_apps" title="Hello World!"></lightning-card>
         <lightning-button label="I'm a button"></lightning-button>
         <xiv-icon-list action-list={mockActionList} job={job}></xiv-icon-list>
+        */</script>
+        <body>
+            <button id="addGCDButton">Add GCD</button>
+            <button id="addWeaveButton">Add Weave</button>
+            <table>
+              <div class="sortable-list">
```
Comment (gik0geck0) [inline src/modules/xiv/rotationBuilder/rotationBuilder.html]: you can inline event binding with a method call like so:
```suggestion
              <div class="sortable-list" ondragstart={dragStart}>
```
Then move the method into the class (the `(e) => ...` portion)

Comment (ThinkingTurtle) [reply src/modules/xiv/rotationBuilder/rotationBuilder.html]: I don't completely understand how the drag events work, should I be adding the inline event binding then adding the functions to the DragEvent class or to the sortable-list class?

**Pull request closed without merging**